networking/spoke-BU0001A0008.json

```
@@ -177,58 +367,11 @@
             }
         },
         {
-            "type": "Microsoft.Network/privateDnsZones",
-            "apiVersion": "2018-09-01",
-            "name": "[variables('acrPrivateDnsZonesName')]",
-            "location": "global",
-            "properties": {
-            }
-        },
-        {
-            "type": "Microsoft.Network/privateDnsZones",
-            "apiVersion": "2018-09-01",
-            "name": "[variables('akvPrivateDnsZonesName')]",
-            "location": "global",
-            "properties": {
-            }
-        },
-        {
-            "type": "Microsoft.Network/privateDnsZones/virtualNetworkLinks",
-            "apiVersion": "2018-09-01",
-            "name": "[concat(variables('acrPrivateDnsZonesName'), '/', uniqueString(resourceId('Microsoft.Network/virtualNetworks', variables('clusterVNetName'))))]",
-            "location": "global",
-            "dependsOn": [
-                "[resourceId('Microsoft.Network/virtualNetworks', variables('clusterVNetName'))]",
-                "[resourceId('Microsoft.Network/privateDnsZones', variables('acrPrivateDnsZonesName'))]"
-            ],
-            "properties": {
-                "virtualNetwork": {
-                    "id": "[resourceId('Microsoft.Network/virtualNetworks', variables('clusterVNetName'))]"
-                },
-                "registrationEnabled": false
-            }
-        },
-        {
-            "type": "Microsoft.Network/privateDnsZones/virtualNetworkLinks",
-            "apiVersion": "2018-09-01",
-            "name": "[concat(variables('akvPrivateDnsZonesName'), '/', uniqueString(resourceId('Microsoft.Network/virtualNetworks', variables('clusterVNetName'))))]",
-            "location": "global",
-            "dependsOn": [
-                "[resourceId('Microsoft.Network/virtualNetworks', variables('clusterVNetName'))]",
-                "[resourceId('Microsoft.Network/privateDnsZones', variables('akvPrivateDnsZonesName'))]"
-            ],
-            "properties": {
-                "virtualNetwork": {
-                    "id": "[resourceId('Microsoft.Network/virtualNetworks', variables('clusterVNetName'))]"
-                },
-                "registrationEnabled": false
-            }
-        },
-        {
-            "apiVersion": "2019-11-01",
             "type": "Microsoft.Network/publicIpAddresses",
-            "name": "[concat('pip-', variables('orgAppId'), '-00')]",
+            "apiVersion": "2020-05-01",
+            "name": "[variables('primaryClusterPipName')]",
             "location": "[parameters('location')]",
+            "comments": "Used as primary entry point for workload. Expected to be assigned to an AAG.",
```

```suggestion
            "comments": "Used as primary entry point for workload. Expected to be assigned to an Azure Application Gateway.",
```
This acronym doesn't seem to be as common, maybe just write it out since we don't use the acronym elsewhere.

---

👍